src/course/handbook/glossary.md

```diff
@@ -0,0 +1,17 @@
+# Glossary or terms
+
+Term | Meaning
+--- | ---
+**argument** | some input to a function
+**attribute** | like a variable, but specifically it's a variable that is defined as part of an *object*
```

Comment:
I'm not sure I've ever heard an object's keys referred to as attributes before. Will it be confusing to people when the language itself has things like `Object.keys()`?